src/e2e-test/features/servicenowmultisource/DesignTimeValidation.feature

```diff
@@ -57,4 +57,23 @@ Feature: ServiceNow Multi Source - Design time validation scenarios
     And Enter input plugin property: "startDate" with value: "start.date"
     And Enter input plugin property: "endDate" with value: "2022-JAN-01"
     And Click on the Validate button
-    Then Verify that the Plugin Property: "endDate" is displaying an in-line error message: "invalid.property.enddate"
\ No newline at end of file
+    Then Verify that the Plugin Property: "endDate" is displaying an in-line error message: "invalid.property.enddate"
+
+  @TS-SN-MULTI-DSGN-ERROR-04
+  Scenario: Verify user should be able to get invalid credentials validation message when using invalid credentials in the connection manager functionality
+    When Open Datafusion Project to configure pipeline
+    And Select plugin: "ServiceNow Multi Source" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "ServiceNow Multi Source"
+    And Click plugin property: "switch-useConnection"
+    And Click on the Browse Connections button
+    And Click on the Add Connection button
+    And Click on the ServiceNow Connector button
+    And Enter input plugin property: "name" with value: "ServiceNowMultiSourceConnection"
```

Comment:
We need unique names for each connection create a test setup hook instead of hardcoding the name.

Reply:
done